test/Experimental/PlaneCurve-test.jl

```diff
@@ -337,12 +337,12 @@ end
     S, (x, y, z) = polynomial_ring(QQ, ["x", "y", "z"])
     T, _ = grade(S)
     C = Oscar.ProjPlaneCurve(T(y^2 * z - x^3 - x * z^2))
-    @test (@inferred Oscar.arithmetic_genus(C)) == 1
-    @test (@inferred Oscar.geometric_genus(C)) == 1
+    @test (Oscar.PlaneCurveModule.arithmetic_genus(C)) == ZZ(1)
+    @test (Oscar.geometric_genus(C)) == ZZ(1)
```

Comment:
With the specification of return values for the various `@attr`, it should be possible to add the `@inferred` back in here. Moreover, it is perfectly fine to compare a `ZZRingElem` to an `Int`. So basically it should be possible to restore this whole test file? I hope?

Reply:
(again: not a blocker, but maybe address in a future PR or at least keep in mind for future changes)